dts/arm/st/stm32f429.dtsi

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/stm32f407.dtsi>
+
+/ {
+	soc {
+		pinctrl: pin-controller {
```

Comment:
you shouldn't need this pinctrl node, should be handled by what's in st/stm32f407.dtsi